docs/geedocs/docsrc/answer/fusionAdministration/commandReference.rst

```diff
@@ -278,6 +277,18 @@ Command reference
       or the configuration fails, the program will return -1 (0 is
       returned on success).
 
+      -\\-chown
+
+      *Optional*. Correct permissions and ownership of the publish root.
+      This is needed when the publish root was created or modified by a user
+      other than the default fusion user, such as when re-using a publish root
```

Comment:
I think in this case the relevant user is the "default Apache user", not the "default fusion user".